src/index.js

```diff
@@ -102,13 +102,23 @@ function mergeExtensionConfig(targetVersion, mergeTarget, extensionConfig, modul
   return deepmerge(mergeTarget, toMerge);
 }
 
+/** @type {string} */
```

Comment:
Type string is wrong. Technically it could be any JSON type (depending on the contents of the `app/etc/hyva-themes.json` file). For that reason, I think `false` is a better default value than an empty string.
In case you don't like `false`, then let's use an empty object, since the configuration usually is a object.